lib/elixir/lib/io/ansi/docs.ex

```diff
@@ -291,6 +291,8 @@ defmodule IO.ANSI.Docs do
     col =
       col
       |> String.replace("\\\|", "|")
+      |> String.replace(~r/-+/, "-")
```

Comment:
Is there a chance this code will replace `foo -- bar` in a column to `foo - bar`?